vm/isolated/isolated.go

```diff
@@ -274,11 +274,11 @@ func (inst *instance) repair() error {
 		} else {
 			log.Logf(2, "isolated: ssh succeeded, trying to reboot by ssh")
 			inst.ssh("reboot") // reboot will return an error, ignore it
+            if err := inst.waitRebootAndSSH(5*60, 30*time.Minute); err != nil {
```

Comment:
This commit broke it right?
https://github.com/google/syzkaller/commit/ddc3e85997efdad885e208db6a98bca86e5dd52f
Before that commit we also did not do waitRebootAndSSH if !TargetReboot.

Reply:
Indeed, it looks like that commit broke the functionality.